rspec-core.gemspec

```diff
@@ -13,6 +13,14 @@ Gem::Specification.new do |s|
   s.summary     = "rspec-core-#{RSpec::Core::Version::STRING}"
   s.description = "BDD for Ruby. RSpec runner and example groups."
 
+  s.metadata = {
+    'bug_tracker_uri'   => 'https://github.com/rspec/rspec-core/issues',
+    'changelog_uri'     => 'https://github.com/rspec/rspec-core/blob/master/Changelog.md',
```

Comment:
This should be the change log from `3-8-maintenance` really, or from a tag..

Reply:
Are you concerned that this would create a lot of maintenance? We would need to update this gemspec file on every minor release.

Reply:
Not necessarily, gem specs are interpreted and we have a constant for the version already...

Reply:
Ah, good thinking. So we could do something like this for a tag:

```ruby
"https://github.com/rspec/rspec-core/blob/v#{s.version}/Changelog.md"
```

The maintenance branch would require a little transformation.